### DISCUSSION TITLE
generate_project.bat only generates CmakeFiles folder and no PhysXSDK.sln

### DISCUSSION THREAD

**Discussion by W4ixr** [NVIDIA-Omniverse/PhysX#121] [Q&A]
I pressed 12 for the solution to have for the Visual Studio 2022 but the .sln files are not present, what I am missing ?

![Unbenannt](https://user-images.githubusercontent.com/127138699/225597710-b7d30722-22ce-40ce-8f96-09355124a658.PNG)

This is the console Output. It says there is no instance running, however i have 2019 installed and open (does it have to be open). I have 2019 version 16.11.25 and also 2022 version 17.3.6

![Screenshot_1](https://user-images.githubusercontent.com/127138699/225598987-06f0c9d9-f079-40d1-8a12-876f82ac432a.png)

**Comment by ayoub-belarbi** [comment]
Hello @W4ixr,
Number 12 is Visual studio 2019 and not 2022.
The error (cmake error) suggest that you don't have visual studio installed so Cmake it's not able to generate the project.
 
 Can you check please if it's installed? Here's some hints how to do that:
 
Check the installation folder: By default, Visual Studio 2019 is installed in the "C:\Program Files\Microsoft Visual Studio\2019". look for the "devenv.exe" file. If it's not there, Visual Studio 2019 is not installed.

You can check also using Visual Studio Installer application (You can see what's installed and what's not)
 
Also There is a possibility that the environment variables are not set. If you have just installed it, maybe you need to reboot? Additionally you can make sure that the environment variables for Visual Studio 2019 are set up correctly by trying to run the "vcvarsall.bat" script to set up the environment variables before running CMake.

Anyway this is a Cmake issue related to your local setup.
Thanks

**Reply by W4ixr** [reply]
Hi, thank you for that fast answer. Its is weird i hav3 2210t installed and i see it in the visual studio installer, however there is no 2019 folder only 2022 at C:\Program Files\Microsoft Visual Studio\. I also rebooted already. Where do i find vcvarsall.bat?


![Unbenannt](https://user-images.githubusercontent.com/127138699/225606404-d9ffff5b-b2e7-4d36-b7cd-8311cbcff4bb.PNG)

**Reply by W4ixr** [reply]
Okay I just realised that my 2019 is installed at C:\Program Files (x86)\Microsoft Visual Studio, does this matter if its in Program Files or just Program Files? And why is the 2022 in the other one?

**Reply by preist-nvidia** [reply]
Hi @W4ixr - we can't help debug cmake / compiler configuration issues, sorry. But you should be able to find answers on Google I would think.

**Reply by W4ixr** [reply]
No worries, hat do reinstall it and tried with the enterprise version instead of community, then it worked. Thank you still.
